lib/cybersource/models/response.rb

```diff
@@ -37,7 +37,7 @@ def self.cybersource_response_params(response)
             :params_cv_code => extract(response, 'cvCode'),
             :params_authorized_date_time => extract(response, 'authorizedDateTime'),
             :params_processor_response => extract(response, 'processorResponse'),
-            :params_reconciliation_id => extract(response, 'reconciliationID'),
+            :params_reconciliation_id => extract(response, 'reconciliationID') || extract(response, 'reconciliation_id'),
```

Comment:
Do we need this? The `response` comes from CyberSource, so it should be camel-cased.